src/System.Data.SqlClient/src/System/Data/SqlClient/SNI/SNIProxy.cs

```diff
@@ -264,8 +264,7 @@ private SNITCPHandle CreateTcpHandle(string fullServerName, long timerExpire, ob
 
             if (serverNamePartsByComma.Length < 2 && serverNamePartsByBackSlash.Length < 2)
             {
-                int defaultInstancePort = TryToGetDefaultInstancePort(fullServerName);
-                return new SNITCPHandle(fullServerName, (defaultInstancePort > 0 ? defaultInstancePort : DefaultSqlServerPort), timerExpire, callbackObject, parallel);
+                return new SNITCPHandle(fullServerName, DefaultSqlServerPort, timerExpire, callbackObject, parallel);
```

Comment:
Are there cases TryToGetDefaultInstancePort would handle that won't be handled by just specifying DefaultSqlServerPort?